### ISSUE TITLE
Configure Unity to work better with git

### ISSUE THREAD

**Issue by asim-shrestha** [asim-shrestha/Showdown#25]
Unity may not be best optimized to work with git at the moment. This ticket will require you do some research on how to make it better and apply these fixes. Some issues right now include hard to read files in pull requests and potentially unwanted files in commits.

Potential changes include:
- improved gitignore
- change project settings

**Comment by seannam1218** [comment]
I agree. Prefabs and gameobjects are a mess when trying to resolve merge conflict
